wp-cli-templates/block-index-js.mustache

```diff
@@ -8,8 +8,9 @@ import { Fragment } from '@wordpress/element';
 /**
```

Comment:
Don't need to import `Path` from `@wordpress/components`, since the icon SVG has been moved.